includes/block/class-chip-gateway.php

```diff
@@ -131,13 +132,10 @@ public function createPayment( Donation $donation, $gatewayData ): RedirectOffsi
 			/* translators: 1: CHIP Checkout URL */
 			give_insert_payment_note( $donation->id, sprintf( __( 'URL: %1$s', 'chip-for-givewp' ), $payment['checkout_url'] ) );
 
-			// $this->log($payment['checkout_url']);
-
 			return new RedirectOffsite( $payment['checkout_url'] );
 		} catch (\Exception $e) {
 			// When debug mode, display details
-			if (WP_DEBUG)
-			{
+			if (WP_DEBUG) {
```

Comment:
`WP_DEBUG` did not necessarily defined. Hence, it is best to check if it is defined first and then check the value.

```
if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
```